src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -90,16 +93,17 @@ function ReportActionItemSingle(props) {
                 style={[styles.alignSelfStart, styles.mr3]}
                 onPressIn={ControlSelection.block}
                 onPressOut={ControlSelection.unblock}
+                disabled={isPolicyAction}
                 onPress={() => showUserDetails(actorAccountID)}
-                accessibilityLabel={actorEmail}
+                accessibilityLabel={isPolicyAction ? displayName : actorEmail}
```

Comment:
NAB I'd rather create methods that return these values instead of adding this `isPolicyAction ? ` logic in the component. That way the logic lives in one place instead of split between the consts above and the prop definition here.

Reply:
fixed.